.github/workflows/jekyll.yml

```diff
@@ -2,10 +2,10 @@ name: Jekyll site CI
 
 on:
   push:
-    branches: [ changes, release, translate ]
+    branches: [ release, next-release ]
   pull_request:
-    branches: [ changes, release, translate ]
-
+    branches: [ release, next-release ]
+  workflow_dispatch:
```

Comment:
What's this?

Reply:
It allows manually triggering the workflow: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch